src/App.tsx

```diff
@@ -1,9 +1,35 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+interface Movie {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+  imdbId: string;
+}
+
 export const App: React.FC = () => {
```

Comment:
you can also use just FC instead of React.FC and import it like { FC } from 'react' )